public/subs.json

```diff
@@ -0,0 +1,2838 @@
+[
```

Comment:
Since you're importing and not fetching, this file can stay inside the component's folder, doesn't need to be in public.